tests/Core/positive/test040.jvc

```diff
@@ -1,6 +1,6 @@
 -- letrec
 
-def sum := letrec sum := \x if x = 0 then 0 else x + sum (x - 1) in sum;
+def sum := letrec sum' := \x if x = 0 then 0 else x + sum' (x - 1) in sum';
```

Comment:
I know it's better not to have variable shadowing, but the point was to test if it works correctly when you do have it.

Reply:
ah, I see. I changed it because it was harder to debug that way. I'll change it back

Reply:
done